test/integration/targets/vmware_guest_register_operation/tasks/main.yml

```diff
@@ -0,0 +1,153 @@
+# Test code for the vmware_guest_register_operation module
+# Copyright: (c) 2019, sky-joker <sky.jokerxx@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Wait for Flask controller to come up online
+  wait_for:
```

Comment:
Please check [`vmware_guest_find`](https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_guest_find/tasks/main.yml) for latest structure of VMware test case.

Reply:
Modified the test case referring to [vmware_guest_find](https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_guest_find/tasks/main.yml).